docs/synapse/devops.rst

```diff
@@ -6,7 +6,63 @@
 Synapse DevOps Guide
 ####################
 
-TBD - See :ref:`quickstart` for current docs.
+General
+=======
+
+TLS/SSL Deployments
+-------------------
+
+For production deployements, it is recommended that all services use the built in ``telepath`` SSL/TLS
+protocol.  You may deploy a service using TLS encryption by specifying a ``--telepath`` listen URL option, such
+as ``ssl://cortex.vertex.link/``.  If it is necessary to separate the socket bind address from the hostname,
```

Comment:
when would this be necessary? (And put that text in)

Reply:
There are a zillion potential reasons for this, not the least of which is him not actually having a public IP address actually assigned to his interface.  I'll make a bit more, but on the whole, the "my bind address does not match my hostname" discussion is out of scope.